mvn-linker/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use indoc::indoc;
+
+use mvn_linker::processor::process;
+use mvn_linker::writer::print;
+
+fn main() {
+    let main_program = indoc! {"
+        1000 0000 ; < ADD_TWO
+        1000 0001 ; < FOO
+        0000 0002 ; > RESULT
+        0000 0004 ;         JP  MAIN
+        0002 0000 ; RESULT  $   /1
+        0004 3003 ; MAIN    LV  /3
+        1006 A000 ;         SC  ADD_TWO
+        1008 9001 ;         MM  FOO
+        000A C000 ;         HM  /0
+        000C 0000 ; # MAIN
+    "};
+    let subroutine_program = indoc! {"
+        6100 0102 ; > ADD_TWO
+        5000 0000 ; < RESULT
+        4100 0002 ; TWO         K   /2
+        4102 0000 ; ADD_TWO     $   /1
+        6104 4100 ;             AD  TWO
+        5106 9000 ;             MM  RESULT
+        6108 B102 ;             RS  ADD_TWO
+    "};
+
+    let complete_linkage = false;
+    let programs = vec![main_program, subroutine_program];
+    let processor = process(programs, complete_linkage);
+    print(processor, complete_linkage);
+}
```

Comment:
Same, coloca na parte de teste e deixa pronto pra ler da stdin

Reply:
Existe uma forma de passar dois arquivos distintos por stdin? Acho que não, mas não sei direito.

Também acho que fazer uma interface só pra esse executável seria excessivo.
Acho que por enquanto vale só ler todos os argumentos como caminhos de arquivos e dar um `panic!` se algum não for. O que você acha?

Reply:
Um desenvolvimento futuro legal que eu pensei agora seria fazer a CLI modular, de forma que desse pra cada módulo definir o seu struct de subcomando e depois a CLI principal unir tudo no `Enum` de subcomandos. Não sei se dá pra fazer, mas vale pesquisar depois.

Reply:
Lembrei que isso não funcionaria ainda porque tem os dois modos do ligador. Acho que a solução seria mesmo separar a CLI pra que cada programa tivesse a sua, que seria depois reutilizada na CLI conjunta

Reply:
Opa, tem que ser via CLI. N me toquei que eram dois arquivos

Reply:
De resto, você acha que não teriam mais outros grandes problemas? Pra mim modificar o binário seria de prioridade bem baixa comparado com mexer no código que em geral vai ser usado com a CLI unificada.

Reply:
Eu experimentei e consegui fazer a definição dos campos da CLI funcionar separado pro linker em https://github.com/PCS3616/mvn-mounter/pull/13/commits/18009306a109782a2382da0075f60e729f3cf4bd